rollup.config.js

```diff
@@ -7,7 +7,6 @@ import replace from 'rollup-plugin-replace'
 export default {
   output: {
     format: 'iife',
-    sourcemap: true
```

Comment:
I was seeing some problems related to it in the watch build - not sure what's up, but I just took it out for now. Since we're not minifying, it's not too hard to trace back the function that's at fault by name. I'll add a ticket to revisit this though.